Mail/Views/New Message/Header Cells/ComposeMessageCellRecipients.swift

```diff
@@ -48,6 +48,9 @@ struct ComposeMessageCellRecipients: View {
 
     @FocusState var focusedField: ComposeViewFieldType?
 
+    @LazyInjectService private var snackbarPresenter: SnackBarPresentable
+    @LazyInjectService private var matomo: MatomoUtils
```

Comment:
Question: why do you prefer to initiate matomo here instead of in the function?

Reply:
As a rule of thumb, I prefer to declare it as an ivar, as it is likely to be used elsewhere in the class / struct.
Indeed here it is used only localy in the function. 

Also if we call 50 times `addNewRecipient`, MatomoUtils will be resolved only once at the init of the class.